mcpy/mcsimulation.py

```diff
@@ -42,54 +42,62 @@ class MCSimulation(object):
         Determines when to log data to std_out, default 10000.
     '''
 
-    def __init__(self, tune_integrators=True, frequency=10000):
+    def __init__(self, beta=1, tune_integrators=True, frequency=10000):
         self._tuning = tune_integrators
+        self.beta = beta
         self.frequency = frequency
         self.step = 0
         self.steps_accepted = []
+        self._integrators = []
         self._log_index = CounterIndex()
 
     def run(self, steps):
         self.check_state()
```

Comment:
I think it would be good if we add some docstrings to here after we finalize there codes.

Reply:
Can be done.